lib/http.js

```diff
@@ -450,6 +451,37 @@ async function _authorize({
   };
 }
 
+/**
+ * Gets an expectedRootCapability from an invocationTarget.
+ *
+ * @param {object} options - Options to use.
+ * @param {string | Array<string>} options.invocationTarget - An
+ *   invocationTarget.
+ * @param {string} options.keystoreId - A url to a key store.
+ *
+ * @return {string | Array<string>} The expectedRootCapability for
+ *   the invocationTarget.
+ */
+function _getExpectedRootCapability({invocationTarget, keystoreId}) {
+  // while no invocationTargets in this module are arrays
+  // it is possible for an invocationTarget to be an array.
+  if(Array.isArray(invocationTarget)) {
+    return invocationTarget;
```

Comment:
@dlongley wouldn't it need to process each item in the array as per the algorithm for and `invocationTarget` that is a string?

Reply:
`invocationTarget` is should always be a string. I'm not sure what's going on where with the docs saying it can be an array. We should fix this so it's always a string.

Reply:
It comes from the `bedrock-edv-storage` one where there are expectedRootCaps that are arrays. Hence the confusion.

Reply:
ok simplified to just a string: https://github.com/digitalbazaar/bedrock-kms-http/pull/14/commits/8b5eb53e0fe3780dcea82786dab1571f7f0b083b